src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -143,6 +151,12 @@ private static void setupLogger(Application application) {
         application.addComponentMenuElement(Logger.class, "Severe level",
                 (ActionEvent e) -> logger.setLevel(Level.SEVERE));
         application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
+
+        // Add loggers to the extensions
+        Extensions.addMenuElementWithCheckbox(application, "Logger", new LoggerListener(DriverFeature.getDriverManager(), loggerDriver));
+        Extensions.addMenuElementWithCheckbox(application, "Tracked Job 2D", new LoggerListener(DriverFeature.getDriverManager(), trackedJob2dDriver));
+        Extensions.addMenuElementWithCheckbox(application, "Precise Logger", new LoggerListener(DriverFeature.getDriverManager(), preciseLoggerDriver));
+        Extensions.addMenuElementWithCheckbox(application, "Precise Logger + Line Drawer", new LoggerListener(DriverFeature.getDriverManager(), loggerAndDrawerContainer));
```

Comment:
The name LoggerListener does not seem to be correct because trackedJob2dDriver also matches the same "extending" pattern as logger drivers.
I suggest adding a separate method in Extensions class for adding such extensions, e.g., 
Extensions.addExtendingDriver("Tracked Job 2D", trackedJob2dDriver);
In this way you would avoid code duplication:
 Extensions.addMenuElementWithCheckbox(application, ...., new LoggerListener(DriverFeature.getDriverManager(), ...));